src/components/icons/Icons-README.md

```diff
@@ -0,0 +1,63 @@
+# Icons
```

Comment:
useful link to preview the rendered markdown: 

https://github.com/getflywheel/local-components/blob/079fcafdeae6f7538943a27c85aa1fd8fd738349/src/components/icons/Icons-README.md

Reply:
very helpful, thanks!